modelcard.md

```diff
@@ -13,23 +14,29 @@ datasets:
 metrics:
 - {metric_0}  # Example: wer. Use metric id from https://hf.co/metrics
 
+# Optional. Add this if you want to encode your eval results in a structured way.
 model-index:
 - name: {model_id}
   results:
-  - task: 
-      type: {task_type}  # Required. Example: automatic-speech-recognition
-      name: {task_name}  # Optional. Example: Speech Recognition
+  - task:
+      type: {task_type}             # Required. Example: automatic-speech-recognition
+      name: {task_name}             # Optional. Example: Speech Recognition
     dataset:
-      type: {dataset_type}  # Required. Example: common_voice. Use dataset id from https://hf.co/datasets
-      name: {dataset_name}  # Optional. Example: Common Voice zh-CN
-      args: {arg_0}         # Optional. Example: zh-CN
+      type: {dataset_type}          # Required. Example: common_voice. Use dataset id from https://hf.co/datasets
+      name: {dataset_name}          # Required. Example: Common Voice (French)
+      config: {dataset_config}      # Optional. Example: fr
```

Comment:
Do we have any doc that documents what config is? I think that would be useful

Reply:
(i think?)

Reply:
Second param is `data_dir` I think https://huggingface.co/docs/datasets/master/en/package_reference/loading_methods#datasets.load_dataset

Reply:
hmm no `name` no?

> name (str, optional) — Defining the name of the dataset configuration

pinging @lhoestq for guidance

Reply:
pretty sure that we also use (or used to use) the `config` name interchangeably on datasets side though

no?

Reply:
Yeah I think in `datasets` we use `name` in the `load_dataset()` and `load_metric()` functions, but `config_name` in `DatasetInfo` and `MetricInfo`. The former is possibly legacy from TFDS: https://www.tensorflow.org/datasets/api_docs/python/tfds/load

Reply:
`name` corresponds to the config name in `load_dataset` indeed.

And "pretty name" in the YAML tags is the name of the full dataset. Here in `name`you may specify the pretty name of the subset (per language for example)